app/scripts/controllers/threebox.js

```diff
@@ -50,6 +50,16 @@ class ThreeBoxController {
           withAppKeyOrigin: 'wallet://3box.metamask.io',
         })
       },
+      processDecryptMessage: (msgParams) => {
```

Comment:
I don't think these two functions are required. This 3Box-specific provider is solely used to facilitate 3Box validating that the box they're creating is authorized by the user - it doesn't use these methods. That's why the `processTypedMessage` functions aren't implemented either, for example.

Reply:
I have removed it. Thank you